docs/src/guides/concepts/hub-search.md

```diff
@@ -0,0 +1,88 @@
+
+The include array takes a list of the "enrichments" to return. e.g `include: ['metadata','server']` In this simple case, before returning the results, the item metadata will be fetched, converted to json, and attached to the `IHubSearchResult` as `.metadata`. For appropriate item types (Feature Service) the `server` information will be fetched and attached as `.server`. Fetching these additional resources takes time, so it's best to do some testing to determine if the information you want is worth the performance hit.
+
+It is also possible to pluck a specific piece of information out of an enrichment and specifying the property name to use on the `IHubSearchResult`. Specifying `include: ["server.layerCount AS numLayers"]` will fetch the server info, then grab just the `layerCount` property and attach that value to the result using the `numLayers` property.
+
+####
+
+| Enrichment  | Entities                           | Description                                               |
+| ----------- | ---------------------------------- | --------------------------------------------------------- |
+| `data`      | `item`                             | Returns the data json for the item.                       |
+| `metadata`  | `item`                             | Returns the item's metadata, in json                      |
+| `groupIds`  | `item`                             | Array of id's of the groups the item is shared to         |
+| `ownerUser` | `item`, `group`                    | returns a full `IUser` for the owner of the group or item |
+| `org`       | `item`                             | returns the `IPortal` of the owning user's org            |
+| `server`    | Feature Service, Map Service items | Returns the feature service definition                    |
+| `layers`    | Feature Service, Map Service item  | Returns the layers array                                  |
+
+### Aggregations
+
+The search can also return aggregation information for the entire result set.
+
+For the most part. the aggregations supported by `hubSearch` are limited to those supported by the ArcGIS Portal Search API. A maximum of three aggregations can be specified.
+
+| Entity  | Aggregation Fields                                      |
+| ------- | ------------------------------------------------------- |
+| `item`  | `tags`, `type`, `access`, `contentstatus`, `categories` |
+| `user`  | none                                                    |
+| `group` | none                                                    |
+
+Once the search is executed, the aggregations are available on the `IHubSearchResponse.aggregations` property as an array of `IHubAggregation` objects.
+
+Applications can use aggregations to construct dynamic facets and filters.
+
+```json
+// Example aggregation stucture
```

Comment:
If you don't have an example structure ready, could you add a todo?

Reply:
I'll be adding to this in the next week